src/client/join.ts

```diff
@@ -42,31 +49,13 @@ export default async function initJoinProcess(params: any) {
 function tryJoinGame(gameID: string, playerName: string) {
   joinGame(gameID)
     .then((res: JoinGameResponse) => {
-      // See if we have a cookie with the token
-      // for this game ID
-      const cookies = document.cookie;
```

Comment:
The meeting token is no longer in the cookie, but in the join response body, hence removing this logic here.